### PR TITLE
chore(EMS-1030): Split map-and-save and save-data for business section

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made:
* Split map-and-save functions in your-business to own files for relevant functions
* Split save-data functions into seperate files and folders for relevant functions
* Updated and split tests

### CODE REVIEW

File: src/ui/server/controllers/insurance/business/map-and-save/contact/index.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      if (!saveResponse) {
+        return false;
+      }
+
+      return true;
+    }
+
+    return true;
+  } catch (err) {
+    console.error('Error mapping and saving business section of application', { err });
```
Comment (ttbarnes) [inline src/ui/server/controllers/insurance/business/map-and-save/contact/index.ts:34]: `business - contact section`?

File: src/ui/server/controllers/insurance/business/map-and-save/company-details/index.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      if (!saveResponse) {
+        return false;
+      }
+
+      return true;
+    }
+
+    return true;
+  } catch (err) {
+    console.error('Error mapping and saving business section of application', { err });
```
Comment (ttbarnes) [inline src/ui/server/controllers/insurance/business/map-and-save/company-details/index.ts:36]: `business - company details section`?

File: src/ui/server/controllers/insurance/business/map-and-save/broker/index.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      if (!saveResponse) {
+        return false;
+      }
+
+      return true;
+    }
+
+    return true;
+  } catch (err) {
+    console.error('Error mapping and saving broker section of application', { err });
```
Comment (ttbarnes) [inline src/ui/server/controllers/insurance/business/map-and-save/broker/index.ts:34]: `business - broker section`?

File: src/ui/server/controllers/insurance/business/save-data/broker/index.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import api from '../../../../../api';
+import getDataToSave from '../../../../../helpers/get-data-to-save';
+import stripEmptyFormFields from '../../../../../helpers/strip-empty-form-fields';
+import { sanitiseData } from '../../../../../helpers/sanitise-data';
+import { Application, RequestBody } from '../../../../../../types';
+
+/**
+ * gets fields to add to the database and sanitises them
+ * saves to broker tables in database via api call
```
Comment (ttbarnes) [inline src/ui/server/controllers/insurance/business/save-data/broker/index.ts:9]: `table` there's only one 😛

File: src/ui/server/controllers/insurance/business/company-details/index.ts [hunk 5]
```diff
@@ -258,7 +258,7 @@ const post = async (req: Request, res: Response) => {
     };
 
     // if no errors, then runs save api call to db
-    const saveResponse = await mapAndSave.companyDetails(updateBody, application);
+    const saveResponse = await companyDetails.mapAndSave(updateBody, application);
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply src/ui/server/controllers/insurance/business/company-details/index.ts:261]: I just thought it makes more sense to be .mapAndSave or .save in the file

**Review by ttbarnes — APPROVED** [review]
There's a typo in this file name but not a blocker, great work!

`save-data/company-details/index.api-test.test.ts`

**Pull request merged**